skbio/sequence/_dna.py

```diff
@@ -148,6 +149,9 @@ class DNA(GrammaredSequence, NucleotideMixin):
 
     """
 
+    read = Read()
```

Comment:
I tent to guess it is not necessary because `Sequence`, which is a parent class of `DNA`, already has `read` and `write` added. Can you test it? Also see my other comment.